sidebars.js

```diff
@@ -68,11 +68,32 @@ module.exports = {
   OperatingSystem: [
     "operating-system",
     "operating-system/getting-started",
+    "operating-system/configuration",
     "operating-system/debugging",
     "operating-system/partition",
     "operating-system/board-metadata",
+    "operating-system/network",
     "operating-system/deployment",
     "operating-system/update-system",
+    {
+      type: "category",
+      label: "Board-specific documentation",
```

Comment:
This makes the sidebar wider then necessary, maybe simply:

```suggestion
      label: "Board support",
```

Reply:
Does it? For me it still fits in the rendered docs in the width of the sidebar, it's the same width as on production.